src/main/java/com/salesforce/storm/spout/dynamic/VirtualSpout.java

```diff
@@ -100,16 +100,10 @@
      */
     private boolean isOpened = false;
 
-    /**
-     * This flag is used to signal for this instance to cleanly stop.
-     * Marked as volatile because currently its accessed via multiple threads.
-     */
-    private volatile boolean requestedStop = false;
-
     /**
      * This flag represents when we have finished consuming all that needs to be consumed.
      */
-    private boolean isCompleted = false;
+    private volatile boolean isCompleted = false;
```

Comment:
Ok with leaving this volatile, but as of right now I don't think we ever read this across threads, so not sure its required.  Perhaps we do in tests.